### ISSUE TITLE
Documentation page - make dynamic?

### ISSUE THREAD

**Issue by brashrebel** [realbigplugins/theme2016#23]
The documentation page is currently a static page I believe. Should we make that an archive that lists all articles? Eventually we'll enhance it with categorization and search but for now there's so little that it should just list the articles.

**Comment by d4mation** [comment]
Well, it is semi-dynamic right now.

If you make a Child Post of your primary Documentation Page it will automatically add it to a floating sidebar that you can skip down to within the page.

They are sorted Alphabetically, so you'll want to name them things like:
```
1. <Title>
2. <Title>
  A. <Title>
  B. <Title>
    i. <Title>
```
With each new tier being a Grandchild Post or Great Grandchild Post.

Here's a screenshot I made earlier that shows off the functionality (In some cases I didn't enter any Content, so you only see the Titles).

We could add an Archive at some point. I believe I have it disabled for the Post Type at the moment as the Single is built around showing all of them at once for that Download.

![image](https://cloud.githubusercontent.com/assets/7770631/17045149/1a5b9226-4f95-11e6-93ed-1d6fea9c404b.png)

**Comment by brashrebel** [comment]
Yeah, I'm mostly talking about this page: https://realbigplugins.com/documentation/ although everything you did and mentioned is awesome. If the main documentation page has nothing then I'll pull it from the menu.

**Comment by d4mation** [comment]
I didn't know we had that page! It must have not made it into my local environment.

I think having searchable documentation would be a good thing to have at some point in the future. Keeping something like that in the Nav would be a good thing. 👍

**Comment by d4mation** [comment]
Update:

While getting things set for LearnDash Gradebook, some changes to Documentation Pages have been made.

1. Child/Grandchild pages are still sorted Alphabetically, but this behavior can be overridden via the `Order` Meta.
2. Child/Grandchild pages now have their Permalinks filtered to go straight to their position in the Top-Most parent Page. This should ultimately make something like Searchable Documentation possible as now in an Archive page clicking a Child Documentation Page would direct you to its location properly. There is no Archive yet, but this is a very positive step toward one.
3. Adding a Link into the Content of a Documentation Page will actually link to its Anchor within the page since I'm filtering everything.
